roles/de-condor-launcher/tasks/update.yml

```diff
@@ -6,7 +6,7 @@
   shell: docker pull discoenv/condor-launcher:{{docker.tag}}
 
 - name: pull configs image
-  shell: docker pull gims.iplantcollaborative.org:5000/de-configs-{{environment_name}}:{{docker.tag}}
+  shell: docker pull {{ docker.registry.base }}/de-configs-{{environment_name}}:{{docker.tag}}
```

Comment:
Since the private Docker registry might be different for different deployments, I wanted to make this configurable. It should make the transition to the new domain name easier too.